src/site/_data/courses/forms/meta.yml

```diff
@@ -0,0 +1,19 @@
+# The date is used to organize the courses on the /learn page.
+# This should be the initial publish date of the course. When new pages are
+# added to the course the site will check if they are <30 days old, and if they
+# are, it will give them a "New" badge. However, when you first launch a course
+# you don't want every entry to have a "New" badge, so anything that existed
+# on or before this date will not receive the "New" badge.
+date: 2021-08-11
+url: /learn/forms/
+draft: true
+title: i18n.courses.learn_forms.title
+description: i18n.courses.learn_forms.description
+# This is the logo that appears in the navigation drawer and header.
+logo: '/images/courses/css/logo.svg'
+# This is used to provide the background for the navigation drawer and header.
+background: '/images/courses/css/background.svg'
+# This provides the cover image for the card on /learn/
```

Comment:
This is kind of interesting - we have images that are clearly meant to be in local `<img>` tags, but some which are meant to be served through imgix. This is probably fine bit a bit ambiguous.

Reply:
Yeah I'm not in love with this. I think it's oddly consistent with the way the site works (all of the /learn/ cards have their svg images in /images) but...it's a bit weird. I didn't think this PR was the right place to change it up tho.

Reply:
Ack